### ISSUE TITLE
TTimeZoneInformation

### ISSUE THREAD

**Issue by limelect** [stella209/PoleCalc#1]
D10.2.3
No TTimeZoneInformation on Android .
On window had to add Winapi.Windows to compile

**Comment by stella209** [comment]
Yes! The CurrentLocalBias function use the TTimeZoneInformatio, but the
Android progran do not use it.
If it is a problem, it may be to be comment or delete this function from
the source code.

Greetings: Agócs László



<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Mentes
a vírusoktól. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

limelect <notifications@github.com> ezt írta (időpont: 2020. okt. 13., K,
20:15):
